IOMC/EventVertexGenerators/src/BetafuncEvtVtxGenerator.cc

```diff
@@ -24,11 +24,8 @@ ________________________________________________________________________
 #include "CLHEP/Random/RandGaussQ.h"
 #include "CLHEP/Units/GlobalSystemOfUnits.h"
 #include "CLHEP/Units/GlobalPhysicalConstants.h"
-//#include "CLHEP/Vector/ThreeVector.h"
```

Comment:
@francescobrivio , is there any reason to keep this commented line?

Reply:
good point, I'll remove it in the next commit

Reply:
ah sorry actually this line **is already removed** in my PR, precisely because it was commented out 😄